include/aos/common/config.hpp

```diff
@@ -314,6 +307,34 @@
 #define AOS_CONFIG_PKCS11_PIN_LEN 42
 #endif
 
+/**
+ * Maximum number of open sessions per PKCS11 library.
+ */
+#ifndef AOS_CONFIG_PKCS11_SESSIONS_PER_LIB
+#define AOS_CONFIG_PKCS11_SESSIONS_PER_LIB 3
+#endif
+
+/**
+ * Maximum number of attributes for object.
+ */
+#ifndef AOS_CONFIG_PKCS11_OBJECT_ATTRIBUTES_COUNT
+#define AOS_CONFIG_PKCS11_OBJECT_ATTRIBUTES_COUNT 10
+#endif
+
+/**
+ * Maximum number of keys per token.
+ */
+#ifndef AOS_CONFIG_PKCS11_TOKEN_KEYS_COUNT
+#define AOS_CONFIG_PKCS11_TOKEN_KEYS_COUNT 20
+#endif
+
+/**
+ * Maximum number of PKCS11 libraries.
+ */
+#ifndef AOS_CONFIG_PKCS11_LIBRARIES_NUM
+#define AOS_CONFIG_PKCS11_LIBRARIES_NUM 1
```

Comment:
[al1img]
set 1 as default. We usually use 1 library.
Author
@[mykola-kobets-epam](https://github.com/mykola-kobets-epam) mykola-kobets-epam [Dec 21, 2023](https://github.com/aoscloud/aos_core_lib_cpp/pull/49#discussion_r1433916483)

fixed